packages/app/src/index.ts

```diff
@@ -1,11 +1,56 @@
 
-import { AeroGearConfiguration, INSTANCE } from "@aerogear/core";
+import find from "lodash.find";
+import console from "loglevel";
+import { AeroGearConfiguration, ServiceConfiguration, MetricsService } from "@aerogear/core";
+
+export class AeroGearApp {
+
+  public metrics?: MetricsService;
+  private configurations?: ServiceConfiguration[];
+
+  /**
+   * @param config configuration that should be injected to all available SDK's
+   */
+  public constructor(config: AeroGearConfiguration) {
+    this.configurations = config.services;
+    const metricsConfig = this.getConfigByType(MetricsService.TYPE);
+    if (metricsConfig) {
+      const metricsOptions = {
+        configuration: this.getConfigByType(MetricsService.TYPE)
+      };
+      this.metrics = new MetricsService(metricsOptions);
+    }
+  }
+
+  /**
+  * Get a service configuration object, provided an existing type is given
+  * @param type - The type of the service
+  */
+  public getConfigByType(type: string): ServiceConfiguration[] | undefined {
+    if (this.configurations) {
+      return this.configurations.filter(service => service.type && service.type.toLowerCase() === type.toLowerCase());
+    }
+  }
+
+  /**
+   * Get a service configuration object, provided an existing id is given
+   * @param id - unique id of the service
+   */
+  public getConfigById(id: string): ServiceConfiguration | undefined {
+    if (this.configurations) {
+      return find(this.configurations, service => {
+        return !!service.id && service.id.toLowerCase() === id.toLowerCase();
+      });
+    }
+    console.error("Configuration not initialized.");
+  }
```

Comment:
if we let each service to use `ServiceConfiguration` in the constructor, then we don't need to duplicate those methods in the `app` module here.